internal/buildscripts/packaging/installer/install.sh

```diff
@@ -290,6 +297,12 @@ ensure_not_installed() {
       exit 1
     fi
   done
+
+  if [ "$with_instrumentation" = "true" ] && [ -f "$instrumentation_so_path" ]; then
+    echo "$instrumentation_so_path already exists which implies that the instrumentation library has already been installed." >&2
+    echo "Please uninstall the instrumentation library and re-run this script" >&2
```

Comment:
not part of these changes, but we may want to evaluate a way to prompt to allow overwriting the existing installation for this and the agents.